ci/pipeline

```diff
@@ -231,6 +234,8 @@ def buildMaster(): Unit = {
   val buildNumber = sys.env.get("BUILD_NUMBER").getOrElse("0")
   val version = build(buildName = s"master-$buildNumber")
   buildDockerAndLinuxPackages()
+  testDockerAndLinuxPackages()
```

Comment:
This seems to be an unrelated change. It will be introduced with #6550.

Reply:
I think this came with the rebase... I remove